<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>PC to Device</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>LED3</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Device to PC</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
        </font>
       </property>
       <property name="text">
        <string>Get LED Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>-10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#00aa7f;&quot;&gt;Device Connected&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>371</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_R">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="redSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="redSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="3">
      <widget class="QWidget" name="displayWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_G">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="greenSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="greenSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_B">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="blueSlider">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="blueSpinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="chartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
